.github/workflows/reusable-build.yml

```
@@ -0,0 +1,450 @@
+          SYSROOT=$(xcrun --sdk macosx --show-sdk-path);
+          export CFLAGS="-isysroot $SYSROOT -isystem $SYSROOT";
+          RUST_TARGET=${{ inputs.target }} pnpm build:binding:${{ inputs.profile }}
+
+      - name: Upload artifact
+        id: upload-artifact
+        uses: ./.github/actions/artifact/upload
+        if: ${{ inputs.target != 'wasm32-wasip1-threads' && steps.check_cache.outputs.exists != 'true' && !inputs.skipable }}
+        with:
+          name: bindings-${{ inputs.target }}
+          path: crates/node_binding/*.node
+          try-local-cache: ${{ inputs.profile == 'ci' }}
```

actionlint (1.7.4)

input "try-local-cache" is not defined in action "Upload Artifact" defined at "./.github/actions/artifact/upload". available inputs are "force-use-github", "name", "path"

(action)